### ISSUE TITLE
Facebook

### ISSUE THREAD

**Issue by Petterholm** [davidmann4/ogar-feeder-bot#341]
Anyone have a way to add multiple fb accounts?

**Comment by Lukkez** [comment]
Really? make one yourself, i already made one, i will don't share, you a **LEECHER**

**Comment by Petterholm** [comment]
can i use this

//auth_token request example

var AgarioClient = require('../agario-client.js'); //Use next line in your
scripts
//var AgarioClient = require('agario-client'); //Use this in your scripts

var account = new AgarioClient.Account();

//Login through facebook on http://agar.io/and copy cookies c_user,datr,xs
from http://www.facebook.com/ here
account.c_user = '1000xxxxxx795';
account.datr = 'q0F5VlxxxxxxxxxxXEJDC';
account.xs = '200:OPXxxxxxxxMw:2:1450787324:-1';

//Request token
account.requestFBToken(function(token, info) {
if(token) {
console.log('Got new token: ' + token);
console.log('Now you can set it in client.auth_token and connect');
console.log('Token will expire in ' + ( account.token_expire-(+new Date) )
- 'ms');
  }else{
  console.log('Failed to get token!');
  if(info.error) console.log('Request error: ' + info.error);
  if(info.res && info.res.statusCode) console.log('HTTP code: ' +
  info.res.statusCode);
  if(info.res && info.res.headers && info.res.headers.location)
  console.log('Redirect: ' + info.res.headers.location);
  if(info.data) console.log('HTML: ' + info.data);
  }
  });

Den søndag 27. mars 2016 skrev Lukkez notifications@github.com følgende:

**Comment by hassanila** [comment]
@Petterholm Yes, but that's not all... that will work for one fb bot only...  you need to make an array with all cookies and get token for all of them then assign them to each bot.

**Comment by Petterholm** [comment]
Can you help me?

Den søndag 27. mars 2016 skrev Hassan notifications@github.com følgende:

**Comment by hassanila** [comment]
@Petterholm I'm busy. som1 else maybe?

**Comment by philw1508** [comment]
Look at stavj Video. He shared fb bots

**Comment by Petterholm** [comment]
Where?

Sendt fra min iPhone
